dcrpm/pidutil.py

```diff
@@ -47,6 +68,43 @@ def pids_holding_file(path):
     return procs
 
 
+def pids_holding_file_lsof(lsof, path):
```

Comment:
given this is just an internal function of `procs_holding_file_lsof` and that it requires you to find `lsof` for it, this should probably be `_pids_holding_file_lsof`